bgpd/bgp_route.c

```diff
@@ -11121,6 +11121,10 @@ static int bgp_show(struct vty *vty, struct bgp *bgp, afi_t afi, safi_t safi,
 		return CMD_WARNING;
 	}
 
+	/* Labeled-unicast routes live in the unicast table. */
+	if (safi == SAFI_LABELED_UNICAST)
```

Comment:
I think we can remove below else if check if we move this on the top? I'm about:

```c
	/* labeled-unicast routes live in the unicast table */
	else if (safi == SAFI_LABELED_UNICAST)
		safi = SAFI_UNICAST;
```

Reply:
yikes - quite right, I'll push that now